src/AppInstallerCLICore/Workflows/WorkflowBase.cpp

```diff
@@ -432,6 +432,15 @@ namespace AppInstaller::CLI::Workflow
         }
     }
 
+    void CheckForMultiQuery(Execution::Context& context)
+    {
+        if (context.Args.GetCount(Execution::Args::Type::MultiQuery) == 1)

```

Comment:
Is there ever a time where the MultiQuery count could be 0? Or where MultiQuery could be null? I'm not sure how all the other parameters and filters like `--id` and `--tag` interact with query.

Reply:
From looking at the implementation of `Args`, it is possible for count to be 0, in that case we also have that `GetArgs(MultiQuery)` is null and `Contains(MultiQuery)` is false.

Other args are just that, other args. You can include only those (like `id`), or you can use only (multi-)query, or you can use both. The interaction happens at the arg validation, or we just pass everything to the search query and let it figure it out.

Reply:
Thanks for the explanation!